.github/workflows/beekeeper.yml

```diff
@@ -14,7 +14,7 @@ jobs:
       REPLICA: 3
       RUN_TYPE: "PR RUN"
       SETUP_CONTRACT_IMAGE_TAG: "0.2.0"
-      BEEKEEPER_BRANCH: "master"
+      BEEKEEPER_BRANCH: "ps-lightnode-fix"
```

Comment:
Shouldn't be this `master` branch?

Reply:
yep, just trying to fix a flaky test with that branch, it will be back to master when merging